webpack/assets/javascripts/foreman_tools.js

```diff
@@ -1,3 +1,7 @@
+'use strict';
+
+const $ = require('jquery');
```

Comment:
This shouldn't be needed as the webpack ProvidePlugin should make $ available to all our modules